wafer/schedule/templates/wafer.schedule/full_schedule.html

```diff
@@ -35,7 +35,7 @@ <h1>
         </div>
       {% endif %}
       {% for schedule_day in schedule_days %}
-        <table cellspacing=1 cellpadding=0>
+        <table cellspacing=1 cellpadding=0 class="table table-striped table-bordered">
```

Comment:
This is probably a better default, but I can still imagine wanting to override it, so it's tempting to do this via SCSS rather than classes.

Are `cellspacing=1 cellpadding=0` still needed with these classes?

Reply:
I have removed that bit for now.